plugins/modules/ec2_instance.py

```diff
@@ -430,6 +430,16 @@
         ebs:
           snapshot_id: snap-1234567890
 
+- name: Create EC2 instance with termination protection turned on 
+    amazon.aws.ec2_instance:
+      name: "my-ec2-instance"
+      vpc_subnet_id: subnet-5calable
+      instance_type: t3.small
+      image_id: ami-123456
+      termination_protection: yes
+      wait: yes
+      count: 1
```

Comment:
```suggestion
  amazon.aws.ec2_instance:
    name: "my-ec2-instance"
    vpc_subnet_id: subnet-5ca1ab1e
    instance_type: t3.small
    image_id: ami-123456
    termination_protection: yes
    wait: yes
```

`count` has some specific implications that aren´t tied to termination protection, I'd avoid it in the example.  `vpc_subnet_id` the random part at the end is hex, so I'd avoid putting `l` in there, the other examples are using `1`.